addons/mail/static/src/discuss/call/common/call_menu.scss

```diff
@@ -1,10 +1,14 @@
 .o-discuss-CallMenu-buttonContent {
     max-width: 150px;
     @include o-mail-call-bordered(.7);
+    height: $o-navbar-height - 20px !important; // same button height as in enterprise.
+    overflow: hidden;
+    color: $o-action !important;
 }
 
 .o-discuss-CallMenu-animation {
     animation: flash 2s;
     animation-direction: alternate;
     animation-iteration-count: 2;
+    transform: translateY(10px);
```

Comment:
Instead of enterprise PR to cancel the community code, you could put a `o-isOdooCommunity`/`o-isOdooEnterprise` modifier classname in templating that relies on `odoo.info.isEnterprise`, so that the SCSS affects only community with only diffing in community code.